### PR TITLE
Revert optimization, breaks re-running. Convert to unix.

### PR DESCRIPTION
**Pull request by costinm**
Sorry about the big diff - but I can't make my editor to use windows line ending, and 
all the files are and should be using \n.

This is urgent, prevents re-running ( when re-run happens, there is no workspace ). In general
workspace can act as a cache, but we should not depend on it. We'll probably need to iterate
few more times - but golang 1.10 may make remove the need for this optimization.

### CODE REVIEW

**Comment by ldemailly** [comment]
blind/on trust
/lgtm

**Comment by ldemailly** [comment]
ideally you'd make 2 PR, one for new lines and then the change you want or at least 2 commits, 1 with only the lf and the other with your actual change so the actual change can be seen/reviewed

**Comment by costinm** [comment]
Yes, ideally we should have small PRs. I tried to avoid dos2unix - but it took to long to find a way to 
edit with \r\n in intellij, and after the mega-PRs that went in it didn't feel so big of a change :-)

To top it off - added a small fix to create the logs directory, something is complaining.

File: .circleci/config.yml [hunk 1]
```diff
@@ -149,7 +149,7 @@ jobs:
       - run: make init go-build test-bins docker.prebuilt
       - run: bin/testEnvRootMinikube.sh wait
       - run: docker images
-      - run: $GOPATH/bin/pilot-integration-test --logtostderr -hub $HUB -tag $TAG -mixer=true -auth=enable -errorlogsdir=/home/circleci/logs -use-sidecar-injector=false
+      - run: $GOPATH/bin/pilot-integration-test --logtostderr -hub $HUB -tag $TAG -mixer=true -auth=enable -errorlogsdir=/home/circleci/logs -use-sidecar-injector=false --core-files-dir=/home/circleci/logs
```
Comment (ldemailly) [inline .circleci/config.yml]: is it possible to add the log level debug or warning flag ?

**Pull request merged**